### DISCUSSION TITLE
Chapter 06. 관계 데이터 연산

### DISCUSSION THREAD

**Discussion by new-pow** [CodeSquad-2023-BE-Study/DB-Study#10] [완료]
책을 읽고 궁금한 점을 남겨주세요.

**Comment by jinny-l** [comment]
처음보는 개념이에요...

![image](https://github.com/CodeSquad-2023-BE-Study/DB-Study/assets/108214590/58df09a9-9108-4a7b-948c-2918a70fd686)
